directory.td.json

```diff
@@ -106,8 +166,8 @@
                 }
             ]
         },
-        "updateTD": {
-            "description": "Update a Thing Description",
+        "retrieveThing": {
+            "description": "Retrieve an individual Thing Description by its ID",
```

Comment:
I think this action is missing the output parameter. Like the events we could just say:
```js
output : {
    type:"object",
    description: "The retrieved Thing Description"
}
```
The full JSON schema of the TD would be better but it super verbose. It is something that we need to improve in the TD spec (e.g., use models to import json schema type definitions). Besides, the content type gives this information so it might be ok to simplify right now.

Reply:
I thought this information was provided by the `contentType`, but I have added the output data schema as well for good measure.